addons/purchase/models/purchase.py

```diff
@@ -101,7 +106,9 @@ def _compute_invoice(self):
     ], string='Billing Status', compute='_get_invoiced', store=True, readonly=True, copy=False, default='no')
 
     # There is no inverse function on purpose since the date may be different on each line
-    date_planned = fields.Datetime(string='Receipt Date', index=True)
+    date_planned = fields.Datetime(string='Receipt Date', index=True,
+                                   help='This is the Receipt Date promised by the supplier. If set, the receipt will be scheduled at this date.')
+    date_calendar_start = fields.Datetime(compute='_compute_date_calendar_start', readonly=True, store=True)
```

Comment:
Maybe an index could be useful for calendar view. Not sure though, maybe to check with stock people.

Reply:
I'll ask and act accordingly